### ISSUE TITLE
How to install

### ISSUE THREAD

**Issue by undermaster** [XaraB/IPAKeyboard#1]
Hello there,

I hope you still answer my newbie-like question. I tried to install your programme, but I didn't manage to get it working.
What I did (not sure about the terms though):
- downloaded the repository via `git clone https://github.com/XaraB/IPAKeyboard.git`
- copied the two folders as said in the readme (I had to create the folder "onboard")
- ran the command "onboard", but it didn't found the command.

My assumption is that I lack a programme called "onboard", but I couldn't find it anywhere (neither in the official debian packages nor on github).

If you could help me with this one, I would really appreciate it, as I find your programme quite useful, especially as you arranged the symbols according to their place of production.
Best,
undermaster

**Comment by XachaB** [comment]
Hi,

I think you're the first user !

Onboard is indeed a program, and what I offer is just an IPA layout & theme for the keyboard. What is your OS ? Onboard is in the ubuntu repos and you can get it by typing in a terminal :

```
sudo apt-get install onboard
```

Maybe I can help more if you tell me more about your system.

**Comment by undermaster** [comment]
Hey, thanks for your quick reply!

My system is Debian 8 Jessie, unfortunately without onboard in the repositories. I already did some research on installing onboard on my system and ran into a lot of problems. I decided to post my issue in another more matching thread (-> https://answers.launchpad.net/onboard/+question/284286). So in case you don't know the answer straight away, I assume I have to figure out this other problem first... Still, thanks a lot for you answer! I'll let you know in case I'll make it.
